lib/double_entry/reporting/aggregate.rb

```diff
@@ -93,16 +95,17 @@ def range_is_complete?
 
       def field_hash
         {
-          :function => function,
-          :account => account,
-          :code => code,
-          :year => range.year,
-          :month => range.month,
-          :week => range.week,
-          :day => range.day,
-          :hour => range.hour,
-          :filter => filter.inspect,
-          :range_type => range.range_type.to_s,
+          :function        => function,
+          :account         => account,
+          :code            => code,
+          :partner_account => partner_account,
```

Comment:
We've resolved this, but for posterity - yes we want to `nil` because to include the aggregates with both `nil` and a `partner_account` would actually be an incorrect number as the aggregates with no partner_account are actually the ones across *all* partner_accounts